tests/system.rs

```diff
@@ -89,13 +104,7 @@ fn test_basic_compile(compiler: Compiler, tempdir: &Path) {
     } = compiler;
     trace!("run_sccache_command_test: {}", name);
     // Compile a source file.
-    // Copy the source files into the tempdir so we can compile with relative paths, since the commandline winds up in the hash key.
-    for f in &[INPUT, INPUT_ERR] {
-        let original_source_file = Path::new(file!()).parent().unwrap().join(f);
-        let source_file = tempdir.join(f);
-        trace!("fs::copy({:?}, {:?})", original_source_file, source_file);
-        fs::copy(&original_source_file, &source_file).unwrap();
-    }
+    copy_to_tempdir(vec![INPUT.to_string(), INPUT_ERR.to_string()], tempdir);
```

Comment:
That way this becomes: `copy_to_tempdir(&[INPUT, INPUT_ERR], tempdir)` and so on.

Reply:
Adapted.